synth.py

```diff
@@ -0,0 +1,29 @@
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+from pathlib import Path
+import subprocess
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+
+versions = ['v1', 'v1p1beta1']
+
+for version in versions:
+    library = gapic.node_library('speech', version)
+
+    # skip index, protos, package.json, and README.md
+    s.copy(
+        library,
+        excludes=['package.json', 'README.md', 'src/index.js'],
+    )
+
+#
+# Node.js specific cleanup
+#
+subprocess.run(['npm', 'install'])
```

Comment:
Let's drop the `prettify and lint` comments, and use `npm ci` instead of `npm install`.  We want to do that so autosynth stops trying to check in automated changes that only have changes to package-lock.json. :)